packages/netlify-cms-media-library-cloudinary/src/index.js

```diff
@@ -74,7 +74,7 @@ async function init({ options = {}, handleInsert } = {}) {
       if (allowMultiple === false) {
         instanceConfig.multiple = false;
       }
-      return mediaLibrary.show({ config: { ...cloudinaryBehaviorConfig, ...instanceConfig } });
```

Comment:
We added `config` on the advice of your team, did the API change?

Reply:
I will double check with the team but adding `config` does not set the configurations for media library.